packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,147 @@ export const Default = {
 };
 
 /**
- * Use the `disabled` attribute to disable an option. Clicks will be suppressed until the disabled state is removed. Use the `checkbox` attribute to prefix a styled checkbox. Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ * Use the `variant` attribute to set the option’s variant.
+ * - `default`
+ * - `checkbox`: Enabled automatically when using `sd-select` with attribute `checklist` set to `true`.
+ *
+ * **Hint:**
```

Comment:
Hint can be deleted since option is now selectable because it is utilized in sd-select.